python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1120,13 +1121,19 @@ def test_parquet_write_partitioned(tmpdir_factory, cols):
     )
     pdf.to_parquet(pdf_dir, index=False, partition_cols=cols)
     gdf = cudf.from_pandas(pdf)
-    gdf.to_parquet(gdf_dir, index=False, partition_cols=cols)
+    gdf.to_parquet(gdf_dir, index=False, partition_cols=cols, partition_file_name=filename)
```

Comment:
Maybe fixes lint

```suggestion
    gdf.to_parquet(
        gdf_dir, index=False, partition_cols=cols, partition_file_name=filename
    )
```

Reply:
Thanks @jakirkham - I just moved to a new environment and forgot to install my pre-commit hooks :)